dapp/src/hooks/useAcrePoints.ts

```diff
@@ -38,8 +38,8 @@ export default function useAcrePoints(): UseAcrePointsReturnType {
       await userPointsDataQuery.refetch()
     },
     onSuccess: (data) => {
-      const claimedAmount = bigIntToUserAmount(data.claimed, 0)
-      const totalAmount = bigIntToUserAmount(data.total, 0)
+      const claimedAmount = Number(data.claimed)
+      const totalAmount = Number(data.total)
```

Comment:
I think that instead of doing two conversion (`string` -> `bigint`-> `number`) we can do it once (`string` -> `number`).

```
    claimed: Number(response.data.claimed),
    total: Number(response.data.total),
```

https://github.com/thesis/acre/blob/f134fc8dca828e02e9520a87258b4fb940bbb714/dapp/src/utils/acreApi.ts#L79-L80

Reply:
Good catch
Ref commit: e66c5263